cmake/BuildParameters.cmake

```diff
@@ -124,14 +124,13 @@ if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "i386")
     #     - Only plugins. No package will link to them.
     set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
 
-    if (DISABLE_ADVANCE_SIMD)
-        set(ARCH_FLAG "-msse -msse2 -march=i686")
-    else()
+    if (NOT DISABLE_ADVANCE_SIMD)
```

Comment:
I see now what i break with that change in generic case.
Pushed new commit - could we wrap those block with "if(NOT DEFINED ARCH_FLAG)" to allow overriding hardcoded values?